app/javascript/app.spec.js

```diff
@@ -1,14 +1,14 @@
-import { shallowMount } from '@vue/test-utils';
-import App from 'app';
+import { shallowMount } from "@vue/test-utils";
+import App from "app";
 
-describe('App', () => {
-  test('is a Vue instance', () => {
+describe("App", () => {
+  test("is a Vue instance", () => {
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'test' is not defined         no-undef